src/Http/Http.Extensions/gen/Resources.resx

```diff
@@ -121,22 +121,22 @@
     <value>Unable to resolve route pattern</value>
   </data>
   <data name="UnableToResolveRoutePattern_Message" xml:space="preserve">
-    <value>Unable to statically resolve route pattern for endpoint. Compile-time endpoint generation will skip this endpoint and the endpoint will be generated at runtime. For more information, please see https://aka.ms/aot-known-issues</value>
+    <value>Unable to statically resolve route pattern for endpoint. Compile-time endpoint generation will skip this endpoint and the endpoint will be generated at runtime. For more information, please see https://aka.ms/rdg-known-issues</value>
```

Comment:
We tend to put our links under the `/aspnet/` segment on aka.ms. [For example.](https://github.com/search?q=repo%3Adotnet%2Faspnetcore%20https%3A%2F%2Faka.ms%2Faspnet%2F&type=code)

Change the link to `https://aka.ms/aspnet/rdg-known-issues`?

Reply:
Good point -- done!